lib/openstudio-standards/standards/Standards.ScheduleCompact.rb

```diff
@@ -1,6 +1,50 @@
 class Standard
   # @!group ScheduleCompact
 
+  # Returns the min and max value in a design day (heating or cooling) from a Compact schedule
+  #
+  # @author Weili Xu, PNNL
+  # @param schedule_compact [OpenStudio::Model::ScheduleCompact] schedule ruleset object
+  # @param type [String] 'heating' will enable the winter design day search, 'cooling' enables summer design day search
+  # @return [Hash] Hash has two keys, min and max.
+  def schedule_compact_design_day_min_max_value(schedule_compact, type = 'heating')
+    vals = []
+    design_day_flag = false
+    prev_str = ''
+    schedule_compact.extensibleGroups.each do |eg|
+      if design_day_flag && prev_str.include?('until')
+        val = eg.getDouble(0)
+        if val.is_initialized
+          vals << val.get
+        end
+      end
+
+      str = eg.getString(0)
+      if str.is_initialized
+        prev_str = str.get.downcase
+        if prev_str.include?('for:')
+          # Process a new day schedule, turn the flag off.
+          design_day_flag = false
+          # in the same line, if there is design day label and matches the type, turn the flag back on.
+          if (prev_str.include?('winterdesignday') && type == 'heating') || (prev_str.include?('summerdesignday') && type == 'cooling') || prev_str.include?('alldays')
```

Comment:
Small suggestion: instead of using `heating` and `cooling` as `type`, perhaps we could use `winter` and `summer` so then that line could be simplified to:
```
if (prev_str.include?(type) || prev_str.include?('alldays')
```

Reply:
Addressed, good suggestion!